Marlin/src/lcd/dogm/HAL_LCD_com_defines.h

```diff
@@ -40,6 +40,13 @@
     #define U8G_COM_HAL_HW_SPI_FN u8g_com_arduino_hw_spi_fn
     uint8_t u8g_com_arduino_st7920_spi_fn(u8g_t *u8g, uint8_t msg, uint8_t arg_val, void *arg_ptr);
     #define U8G_COM_ST7920_HAL_SW_SPI u8g_com_arduino_st7920_spi_fn
+  #elif defined(ARDUINO_ARCH_STM32)
+    uint8_t u8g_com_arduino_std_sw_spi_fn(u8g_t *u8g, uint8_t msg, uint8_t arg_val, void *arg_ptr);
+    #define U8G_COM_HAL_SW_SPI_FN u8g_com_arduino_std_sw_spi_fn    
```

Comment:
indeed, in U8glib-HAL_ID1932/src/clib/u8g_com_arduino_std_sw_spi.c

Reply:
It's a good idea, but I have no STM32F1xx board to verify this.

Reply:
yep indeed, in fact it does not work on the SKRmini... spi mode thing i think

Reply:
> it does not work on the SKRmini

The above change? Or "deleting the HAL file" ?